dist/react-select.js

```diff
@@ -1138,6 +1141,20 @@ var Value = React.createClass({
 		}
 	},
 
+	getInitialState: function getInitialState() {
+		return {
+			popOverOpen: false
```

Comment:
This was already merged and this is super minor but Bootstrap docs refer to popover as `Popover` so the casing should be: `popoverOpen` IMO.